### PR TITLE
Update FAQ answer on deployment

### PR DESCRIPTION
**Pull request by timdebruin**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -75,6 +75,4 @@ No, currently we are not planning on releasing a `PyTorch` version of Larq.
 
 ## Can I use Larq for the deployment of my models?
 
-Currently, Larq is designed purely for training BNNs and there is no support for deployment of binarized models. This means that at this moment Larq is most useful to researchers and less so for developers working on applications.
-
-Of course, the real goal of BNNs is efficient deployment, and in the future we will offer solutions for smooth deployment of models created and trained with Larq.
+Absolutely! For this purpose We have [Larq Compute Engine (LCE)](https://docs.larq.dev/compute-engine/): a highly optimized inference engine for deploying extremely quantized neural networks, such as Binarized Neural Networks (BNNs). The LCE works seamlessly with the rest of Larq.
```
Comment (AdamHillier) [inline docs/faq.md:78]:
```suggestion
Absolutely! For this purpose we have [Larq Compute Engine (LCE)](https://docs.larq.dev/compute-engine/): a highly optimized inference engine for deploying extremely quantized neural networks, such as Binarized Neural Networks (BNNs). The LCE works seamlessly with the rest of Larq.
```

Comment (AdamHillier) [reply docs/faq.md]: As a general question, do we refer to it with a definite article as 'The LCE' or just 'LCE' without a definite article as 'LCE'?

File: docs/faq.md [hunk 2]
```diff
@@ -75,6 +75,4 @@ No, currently we are not planning on releasing a `PyTorch` version of Larq.
 
 ## Can I use Larq for the deployment of my models?
 
-Currently, Larq is designed purely for training BNNs and there is no support for deployment of binarized models. This means that at this moment Larq is most useful to researchers and less so for developers working on applications.
-
-Of course, the real goal of BNNs is efficient deployment, and in the future we will offer solutions for smooth deployment of models created and trained with Larq.
+Absolutely! For this purpose we have [Larq Compute Engine (LCE)](https://docs.larq.dev/compute-engine/): a highly optimized inference engine for deploying extremely quantized neural networks, such as Binarized Neural Networks (BNNs). The LCE works seamlessly with the rest of Larq.
```
Comment (lgeiger) [inline docs/faq.md:78]: For linking here to work you don't need to have the full URL:
```suggestion
Absolutely! For this purpose we have [Larq Compute Engine (LCE)](/compute-engine/): a highly optimized inference engine for deploying extremely quantized neural networks, such as Binarized Neural Networks (BNNs). The LCE works seamlessly with the rest of Larq.
```

**Comment by AdamHillier** [comment]
Do we refer to the compute engine as 'LCE' or 'The LCE' as a general question?

**Comment by timdebruin** [comment]
> Do we refer to the compute engine as 'LCE' or 'The LCE' as a general question?

LCE (although under strong objection of me) I now see that my objection has seeped through into my PR, will rectify that.

File: docs/faq.md [hunk 3]
```diff
@@ -75,6 +75,4 @@ No, currently we are not planning on releasing a `PyTorch` version of Larq.
 
 ## Can I use Larq for the deployment of my models?
 
-Currently, Larq is designed purely for training BNNs and there is no support for deployment of binarized models. This means that at this moment Larq is most useful to researchers and less so for developers working on applications.
-
-Of course, the real goal of BNNs is efficient deployment, and in the future we will offer solutions for smooth deployment of models created and trained with Larq.
+Absolutely! For this purpose we have [Larq Compute Engine (LCE)](/compute-engine/): a highly optimized inference engine for deploying extremely quantized neural networks, such as Binarized Neural Networks (BNNs). The LCE works seamlessly with the rest of Larq.
```
Comment (timdebruin) [inline docs/faq.md:78]:
```suggestion
Absolutely! For this purpose we have [Larq Compute Engine (LCE)](/compute-engine/): a highly optimized inference engine for deploying extremely quantized neural networks, such as Binarized Neural Networks (BNNs). LCE works seamlessly with the rest of Larq.
```

**Pull request merged**